apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx

```diff
@@ -25,33 +25,32 @@ export const FirebaseRemoteConfigProvider = ({ children }: Props): JSX.Element |
       console.error(`Failed to initialize Firebase SDK with error: ${error}`);
       setLoaded(true);
     }
+    const remoteConfig = getRemoteConfig();
+    remoteConfig.settings.minimumFetchIntervalMillis = 0;
```

Comment:
Is it safe to set this to 0 even in prod?

Reply:
yes, it is very safe.
even if the firebase server is not accessible or is blocked due to the rate limit, the "getValue" method can get the config from local cache(if the local cache fails, it gets config from default value)